dist/index.js

```diff
@@ -3784,6 +3784,45 @@ exports.request = request;
 //# sourceMappingURL=index.js.map
 
 
+/***/ }),
+
+/***/ 2183:
+/***/ ((module) => {
+
+"use strict";
+
+
+/**
+ * Returns warning message if the action reference is pinned to master/main
+ *
+ * @param     repoName          Full name of the repo (owner/repo-name)
+ * @return   { String | null }  Warning to be emitted
+ */
+function getActionRefWarning(repoName) {
+  if (!repoName) return null
+
+  const actionRef = process.env.GITHUB_ACTION_REF
+
+  if (actionRef === 'main' || actionRef === 'master') {
+    return (
+      `${repoName} is pinned at HEAD. We strongly ` +
+      `advise against pinning to "@master" as it may be unstable. Please ` +
+      `update your GitHub Action YAML from:\n\n` +
+      `    uses: '${repoName}@master'\n\n` +
```

Comment:
Maybe replace 
from `uses: '${repoName}@master'\n\n`
to:  `uses: '${repoName}@${actionRef}'\n\n`
The message will be more accurate with the current `actionRef`